README.md

```diff
@@ -1,6 +1,53 @@
 Bowling Challenge in Ruby
 =================
 
+#### Makers Week 5 Weekend Challenge - bowling scorecard.
```

Comment:
Very clear README. Maybe could be good to add snips of flow chart/domain modelling.